protocols/readme.md

```diff
@@ -0,0 +1,30 @@
+This package currently contains two prototype implementations of the direct funding protocol.
```

Comment:
This file also should be tidied up a bit.

Reply:
Optional: we could record the way we *aren't* doing it (and why) in an architectural decision record md file in this directory?